### ISSUE TITLE
check and clean the trained keras_lstm models

### ISSUE THREAD

**Issue by schwittlick** [Schwittleymani/ECO#118]
and check logs for models with smallest loss result.

```
/projects/eco/src/python/keras_lstm/logs/arthur_danto-the_end_of_art_a_philosophical_defence_i10e10s2.log
```

shows iteration 3 has lowest loss of 0.8

loss of iteration 10 is 2.4 (worse than iteration 3)

i dont really know what the loss is. maybe gotta read this http://deeplearning4j.org/lstm.html
